pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -3875,6 +3875,204 @@ def prepare_plots(self):
                 }
 
 
+class DriveCrosstalkCancellationAnalysis(MultiQubit_TimeDomain_Analysis):
+    def process_data(self):
+        super().process_data()
+
+        pdd = self.proc_data_dict
+
+        pdd['ramsey_phases'] = np.unique(
+            self.get_param_value('sweep_points')[0]['phase'][0])
+        pdd['qb_sweep_points'] = {}
+        pdd['qb_sweep_param'] = {}
+        for k, v in self.get_param_value('sweep_points')[0].items():
+            if k == 'phase':
+                continue
+            qb, param = k.split('.')
+            pdd['qb_sweep_points'][qb] = v[0][::len(pdd['ramsey_phases'])]
+            pdd['qb_sweep_param'][qb] = (param, v[1], v[2])
```

Comment:
I will try to fix the above lines that handle SweepPoints after I fix their creation in the measure_function. Ideally, these lines can be completely removed in that case.

Reply:
Fixed now.